sway/tree/workspace.c

```diff
@@ -258,7 +258,7 @@ struct sway_container *workspace_by_name(const char *name) {
 	struct sway_container *current_workspace = NULL, *current_output = NULL;
 	struct sway_container *focus = seat_get_focus(seat);
 	if (focus) {
-		current_workspace = container_parent(focus, C_WORKSPACE);
+		current_workspace = focus->type == C_WORKSPACE ? focus : container_parent(focus, C_WORKSPACE);
```

Comment:
We have an 80 character line limit, so this line needs to be wrapped onto two. That 80 characters assumes a tab width of 4 by the way.

Reply:
@RyanDwyer would wrapping on the `?` be acceptable here?

Reply:
Yep, that's perfect.